packages/core/src/login.ts

```diff
@@ -128,13 +153,64 @@ export const login = async (dataDir: string, loginUrl: string, lcUrl: string, cl
 
   const postLoginResponse = await fetch(
     `${lcUrl}/api/cli/post-login`,
-    { method: 'POST',
+    {
+      method: 'POST',
       body: JSON.stringify({ id_token: tokens.id_token }),
-      headers: { 'Content-Type': 'application/json', Authorization: `Bearer ${tokens.access_token}` } }
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${tokens.access_token}`,
+      },
+    }
   )
 
   if (postLoginResponse.ok) {
-    logger.info(`Logged in successfully as: ${jose.decodeJwt(tokens.id_token).email} 👌`)
+    const postLoginData: PostLoginResult = await postLoginResponse.json()
+    if (!('currentOrg' in postLoginData)) {
+      const {
+        orgName,
+        associateDomain,
+        // eslint-disable-next-line no-use-before-define
+      } = await inquirer.prompt<{ orgName: string; associateDomain: string }>([
+        {
+          type: 'input',
+          name: 'orgName',
+          message: 'Select name for your organization',
+          default: postLoginData.organizationDomainDetails.name,
+        },
+        {
+          type: 'confirm',
+          name: 'associateDomain',
+          message: `Allow anyone with ${postLoginData.organizationDomainDetails.domain || 'your'} email address to join your organization as viewers`,
```

Comment:
The phrasing `Allow anyone with your email address to join your organization` isn't clear. What does `your email address` mean here? Also - is it *allowing* (e.g, with some kind of confirmation) or *automatic joining*?

Reply:
currently it's automatic joining.
`Allow anyone with **livecycle.io** email address to join your organization.`
the 'your' part is just a default value that will probably never happen. With the actual domain, isn't the phrase clear enough?